src/main/scala/com/nec/ve/VeProcess.scala

```diff
@@ -181,8 +181,12 @@ object VeProcess {
         s"Expected > 0, but got ${functionAddr} when looking up function '${functionName}' in $libraryReference"
       )
 
+      val start = System.currentTimeMillis()
```

Comment:
Should extract a method for this. Eg `pushMetricTime(_.registerVeCall) { /* ... */ }`

Reply:
Any chance to also do these in the VE plans? Ie per each plan to have a timer for the ve calls; we might find some slownesses in individual plan executions.

Reply:
Yeah, I wanted to do it quickly so that's why I decided not to make methods but sure can be done :)